assets/scss/app.scss

```diff
@@ -219,7 +219,7 @@ pre.looped-records {
 }
 
 .govuk-radios__item .govuk-hint {
-  margin-top: 0rem;
+  margin-top: 5px;
```

Comment:
This will affect all pages with radio options. Is this something we want or this should be specific to this page?

Reply:
As discussed this looks fine as per figma.